src/drivers/github.test.js

```diff
@@ -1,4 +1,4 @@
-jest.setTimeout(20000);
+jest.setTimeout(40000);
```

Comment:
Had to increase all these timeouts for tests to run correctly.

Reply:
odd. CML+=40, BB+=100, GH+=20. No clear pattern.

Reply:
I adjusted them haphazardly to avoid spamming subscribed users with more comments. The values I chose work well enough, but we can run a binary tree search or set in place some profiling to find the optimum values if you want to.

Reply:
not a massive priority - can revisit if timeouts are actually being hit.

Reply:
Agreed! By the way, the Bitbucket timeout is so high because we're running not one but two paginated API queries, each with, literally, a thousand comments from earlier tests. That's the issue with not using ephemeral testing repositories.